components/atoms/a-images-grid.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="a-images-grid">
+    <div v-for="singleRow in row" :key="singleRow" class="a-images-grid__row">
+      <div v-for="singleCol in col" :key="singleCol" class="a-images-grid__col">
+        <SfImage
+          :src="getImage(singleRow, singleCol)"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import {
+  SfImage
+} from '@storefront-ui/vue';
+export default {
+  name: 'AImagesGrid',
+  props: {
+    row: {
+      type: Number,
+      default: 2,
+      validator: (value) => value > 0
+    },
+    images: {
+      type: Array,
+      default: () => []
+    }
+  },
+  components: {
+    SfImage
+  },
+  computed: {
+    col () {
+      const images = this.images.length || 1
```

Comment:
Why ` || 1` is needed here? I think that if there are no images then nothing should be shown 🤔 
This ` || 1` can break the `getImage` method if `this.images` is an empty array - `getImage` will then return `undefined` as a result from `this.images[0]`.

Reply:
yep, thats true, I removed it. Now if there is col === 0 then it will not render image which is what we want :+1: